.github/workflows/claude-nl-suite.yml

```
@@ -0,0 +1,290 @@
+          PY
+
+      - name: Fallback JUnit if missing
+        if: always()
+        run: |
+          set -eu
+          mkdir -p reports
+          if ! ls reports/junit-*.xml >/dev/null 2>&1; then
+            echo "No JUnit found; writing fallback."
+            {
+              cat <<'XML'
+<?xml version="1.0" encoding="UTF-8"?>
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 253: could not find expected ':'

(syntax-check)